build/Targets/Settings.props

```diff
@@ -71,6 +71,9 @@
 
     <RoslynNetSdkRootPath>$(MSBuildSDKsPath)\Microsoft.NET.Sdk\Sdk\</RoslynNetSdkRootPath> 
     <RestoreProjectStyle>PackageReference</RestoreProjectStyle>
+
+    <!-- We should be able to remove this extra property once Roslyn projects include the Sdk="Microsoft.NET.Sdk" attribute. -->
```

Comment:
Sorry, I should have said "Not importing Microsoft.Common*.targets before setting the properties we define in Imports.targets has subtle effects..."

A concrete example is that when you use the ```<Project SDK=...``` syntax, ```Microsoft.Common*.targets``` will get imported ***after*** our Imports.targets and will squat on the values set for ```CoreCompileDependsOn```.  This can seemingly rewritten by using BeforeTargets/AfterTargets (so we'll call the IVT and AssemblyInfo generators before CoreCompile), but there are no doubt subtle differences in expressing it the original way vs the latter.